notebooks/002-openvino-api/002-openvino-api.ipynb

```diff
@@ -329,14 +330,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from openvino.inference_engine import IECore\n",
+    "from openvino.runtime import Core\n",
     "\n",
-    "ie = IECore()\n",
+    "ie = Core()\n",
     "classification_model_xml = \"model/classification.xml\"\n",
-    "net = ie.read_network(model=classification_model_xml)\n",
-    "exec_net = ie.load_network(network=net, device_name=\"CPU\")\n",
-    "input_layer = next(iter(net.input_info))\n",
-    "output_layer = next(iter(net.outputs))"
+    "model = ie.read_model(model=classification_model_xml)\n",
+    "compiled_model = ie.compile_model(model=model, device_name=\"CPU\")\n",
+    "input_layer = next(iter(compiled_model.inputs))\n",
+    "output_layer = next(iter(compiled_model.outputs))"
```

Comment:
![image](https://user-images.githubusercontent.com/10940214/155842408-6d602791-55b0-4116-a822-e26a01a8dedf.png)
Check the markdown cell, model.outputs no net.outputs.